model/src/main/proto/spine_examples/chatspn/message/removal_rejections.proto

```diff
@@ -67,4 +67,7 @@ message MessageCannotBeMarkedAsDeleted {
 
     // The ID of the user who originally asked to mark this message as deleted.
     spine.core.UserId user = 3 [(required) = true];
+
+    // The ID of the process that originally told to mark the message as deleted.
+    MessageRemovalOperationId process  = 4 [(required) = true];
```

Comment:
It's probably `operation` (so is the term in its documentation). The reason here is that `MessageRemovalOperationId` is an _operation_ ID.